vinca_linux_64.yaml

```diff
@@ -37,111 +37,114 @@ skip_existing:
 packages_select_by_deps:
   # only subset of packages to reduce maintainer load
   # trigger
-  - qt-qui-cpp # needs to be compiled locally
```

Comment:
It's a bit hard to keep track of the package list with the re-ordering, but are we rebuilding all packages that had been built previously?

Reply:
I started from the existing PR so I did not changed a lot that. To be honest, I did not checked this, but the build indicate that ~650 package have been built, while according to https://robostack.github.io/humble.html in the past we built 625 . I do not know anyhow if the 625 already built packages are a subset of the 650 newly build packages, I can do some quick checks. 

Anyhow, is the new list "structure" ok for you or do you prefer that we go back to the old one?

Reply:
The new list structure is good, thanks :). And I guess knowing that we build approximately the same number of packages as before is good enough, end-users can raise an issue if a package is missing.